haystack/nodes/prompt/prompt_node.py

```diff
@@ -293,66 +476,95 @@ def get_predefined_prompt_templates() -> List[PromptTemplate]:
     return [
         PromptTemplate(
             name="question-answering",
-            prompt_text="Given the context please answer the question. Context: $documents; Question: "
-            "$questions; Answer:",
+            prompt_text="Given the context please answer the question. Context: {join(documents)}; Question: "
+            "{query}; Answer:",
+            output_shapers=[
+                Shaper(
+                    func="strings_to_answers",
+                    outputs=["answers"],
+                    inputs={"strings": "results", "prompt": "prompt", "documents": "documents"},
+                )
+            ],
+        ),
+        PromptTemplate(
```

Comment:
The previous impl of the question-answering template that runs one prompt per doc is now available under a different name